### PR TITLE
Address dependabot alerts

### PR DESCRIPTION
**Pull request by smudge**
Committing the lockfile triggered a bunch of dependabot errors, and I realized that the versions committed were very out of date, so this updates the lockfile with newer versions.

/no-platform

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -126,4 +206,4 @@ DEPENDENCIES
   zeitwerk
 
 BUNDLED WITH
-   2.4.13
+   2.4.11
```
Comment (effron) [inline Gemfile.lock:209]: should we avoid decreasing the bundler version?

**Review by effron — DISMISSED** [review]
just one nonblocking comment

domainLGTM!

**Review by Irving-Betterment — APPROVED** [review]
domainlgtm! Other than the bundler version Harry pointed out.

**Review by effron — APPROVED** [review]
domainLGTM

**Pull request merged**